packages/react-cosmos-component-playground/src/components/DisplayScreen/__tests__/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Loader } from 'react-cosmos-loader';
+import DisplayScreen from '../';
+import hiThereFixture from '../__fixtures__/hi-there';
+
+const shallowLoader = element =>
+  shallow(element)
+    .dive() // Loader
+    .dive(); // PropsProxy
+
+describe('DisplayScreen', () => {
+  let wrapper;
+
+  beforeAll(() => {
+    wrapper = shallowLoader(
+      <Loader component={DisplayScreen} fixture={hiThereFixture} />
```

Comment:
I don't understand why did you chose to render `DisplayScreen` through 'react-cosmos-loader' (and maybe also other components in this PR). The `DisplayScreen` is a pretty basic component independent of the loader and this test will fail if someone breaks the loader. Maybe I'm missing something though.